Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml

```diff
@@ -37,3 +37,127 @@
     sprite: DeltaV/Structures/Walls/railing.rsi
   - type: Icon
     sprite: DeltaV/Structures/Walls/railing.rsi
+
+- type: entity
+  parent: Railing
+  id: RailingDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+
+- type: entity
+  parent: RailingCorner
+  id: RailingCornerDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+
+- type: entity
+  parent: RailingCornerSmall
+  id: RailingCornerSmallDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+
+- type: entity
+  parent: RailingRound
+  id: RailingRoundDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+
+- type: entity
+  id: BaseRailingWood
+  suffix: Modern, Wood
+  components:
```

Comment:
add `abstract: true` above id so this cant be spawned directly

Reply:
👍